packages/application-shell/src/components/application-shell-provider/application-shell-provider.js

```diff
@@ -61,47 +60,32 @@ export default class ApplicationShellProvider extends React.Component {
                 <GtmBooter
                   trackingEventWhitelist={this.props.trackingEventWhitelist}
                 >
-                  <Switch>
-                    {/**
-                     * No matter if the user is authenticated or not, when we go
-                     * to this route we should always log the user out.
-                     * TODO: do a hard page reload once we move the login routes
-                     * to a different app.
-                     */}
-                    <Route path="/logout" component={Logout} />
```

Comment:
This needs to stay and to do a hard redirect. Unless we change the `logout` link to be a hard redirect

Reply:
I added a hard-redirect in the same vein as in the other component now.